src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -89,78 +89,47 @@ def on_simulation_end(self, event: Event) -> None:
         self.responses.to_hdf(self.output_dir / self.output_filename, key="responses")
 
 
-# FIXME: give this a more descriptive name (eg CensusObserver)
-class Observers:
-    # FIXME: the docstring is no longer correct; update it
+class CensusObserver(BaseObserver):
```

Comment:
The intention is to have a single Observer for the census, not a separate component for each time we want to take the census.

Reply:
I think it will be simpler to include a separate observer for each decade.  Let me see if I can convince you.  The key points are (1) it will simplify the logic of testing if the timestep is right for observation; and (2) it will make it easier to have multiple censuses for testing purposes without running for a full decade.  Let's discuss in real time if necessary.

Reply:
But before I forget (3) if we have a single observer we have to (a) put all years in a single table or (b) extend the BaseObserver to allow (b1) saving multiple keys or (b2) saving multiple files, and I think these tables are big enough that having them in separate files is better than options (a) or (b1).  This approach accomplishes something equivalent to (b2) without requiring changes to BaseObserver.

Reply:
Rajan strongly prefers an approach with a single observer, so that we can change the duration of the sim and get all the decennial censuses observed without having to add additional CensusObserver lines to the model spec, which I have implemented in https://github.com/ihmeuw/vivarium_census_prl_synth_pop/commit/e6c49f7f4e0a888f329bf362481af84b110032a0